rts/lra/client-jakarta/pom.xml

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2022 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is it necessary to provide the files in this pull request under ASL 2?

Reply:
@tomjenkinson The non-jakarta version that I based this pom on had it in and I don't know what to do about that.

So I have pushed a version that has no copyrights since it matches the majority of the LRA module poms.

Reply:
Thanks for the response - if you based it on some other poms in the repo then I think it would have the same header